.gitignore

```diff
@@ -8,3 +8,6 @@ src/.vscode
 src/data/*.ini
 src/data/*.json
 src/hardware
+src/VERSION
+src/firmware.bin.gz
+src/user_defines.txt
```

Comment:
This one should not be ignored. What should actually happen is that Configurator should create/modify `super_defines.txt` instead, which is ignored.

Reply:
Yeah, okay, it's strange that "VERSION" is included in .gitignore, I removed it. 

However, when building firmware for Happymodel 900 MHz from the local directory, it appears that the "user_defines.txt" file gets modified. After looking at the code of ExpressLRS-Configurator, I found that the changes to "user_defines.txt" occur here: https://github.com/ExpressLRS/ExpressLRS-Configurator/blob/master/src/api/src/library/FirmwareBuilder/index.ts#L84. If this file is indeed being modified, I believe it should be included in .gitignore.

Additionally, when building firmware for the Happymodel ES900 RX, a "firmware.bin.gz" file is generated in the "src" directory. This seems to be a bug, as the firmware.bin.gz file should only be present in the temporary directory where build artifacts are stored, and not in the src directory. Therefore, I believe it should not appear in the src directory. If this is the case, I will remove it from the .gitignore file and attempt to resolve the issue.